src/main/java/frc/team2412/robot/util/auto/AutoLogic.java

```diff
@@ -325,6 +293,14 @@ public static boolean chooserHasAutoSelected() {
 	}
 
 	public static Command getSelectedAuto() {
+
+		// TODO: figure out why code crashes when this sequential command group is returned
+		// code also crashes when i run without wait command, something about composition error? idk
```

Comment:
It isn't allowed to use the same command in multiple compositions. Generally, this is avoided by recreating commands each time, which in this case probably looks like making `AutoPath` create the command in `getAutoCommand()` instead of making it once in the constructor and reusing it.

Reply:
It would be useful for drive team to be able to set a delay timer before an auto starts without programming having to add the delay via hardcode/pathplanner. How would you suggest getting around said error?

Reply:
What I just said- Change `AutoPath.getAutoCommand()` to `return AutoBuilder.buildAuto(pathPlannerAutoName)` and remove `autoCommand`.